service/controller/v19/resource/ipam/create.go

```diff
@@ -39,15 +39,11 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		//
 		//     https://github.com/giantswarm/giantswarm/issues/4192
 		//
-		if key.CIDR(customObject) != "" {
```

Comment:
I think we should check for `deprecated` string explicitly. CIDR in wrong format may be caused by other problems in the stack.

FYI this is how it was done before https://github.com/giantswarm/kubernetesd/commit/4d8d9efcbfd4b621c4eb916c343ed08fab0e6645#diff-6ca759f68647d79bdade44f011a47ff9R138

I don't mind it being in upper case though.